app/app.tsx

```diff
@@ -87,6 +91,61 @@ const CustomToast = () => {
   return <Toast config={toastConfig} topOffset={insets.top} />
 }
 
+// Setting up our OTA Updates component
+const OTAUpdates = () => {
```

Comment:
Not critical. Would be nice to have this in it's own file to keep app.tsx more clean.

Reply:
agreed. will be refactored in my other PR #189